<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>378</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>553</width>
    <height>378</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>553</width>
     <height>378</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>535</width>
        <height>360</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <widget class="QLabel" name="label_0">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>WD电商交易系统</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="pwdLineEdit_0">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="userLineEdit_0">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="loginBtn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>70</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QLabel" name="userLabel_0">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>用户名：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="pwdLabel_0">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>密码：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="quitBtn">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>220</y>
          <width>70</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
       <widget class="QPushButton" name="registerNowBtn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>270</y>
          <width>201</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>没有账号？ 点我注册</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="registerPage">
       <widget class="QLabel" name="userLabel_1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>用户名：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_1">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>70</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>WD电商交易系统</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="pwdLineEdit_1">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="pwdLabel_1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>密码：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="userLineEdit_1">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="backBtn">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>260</y>
          <width>70</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QPushButton" name="registerBtn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>260</y>
          <width>70</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="repaetPwdLineEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>210</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="repeatPwdLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>210</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>重复密码：</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="mainPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userLineEdit_0</tabstop>
  <tabstop>pwdLineEdit_0</tabstop>
  <tabstop>loginBtn</tabstop>
  <tabstop>registerNowBtn</tabstop>
  <tabstop>quitBtn</tabstop>
  <tabstop>userLineEdit_1</tabstop>
  <tabstop>pwdLineEdit_1</tabstop>
  <tabstop>repaetPwdLineEdit</tabstop>
  <tabstop>registerBtn</tabstop>
  <tabstop>backBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
